src/components/form-template/form-template.component.tsx

```diff
@@ -2,6 +2,8 @@ import React, { useState, useEffect } from 'react';
 import cx from 'classnames';
 import { Collapse } from 'reactstrap';
 
+import { ReactComponent as ErrorIcon } from '../../assets/img/icon-alert-danger-md.svg';
```

Comment:
Merk to ting her: default import og SVG hentet fra images mappa (img er legacy og har sære regler for import)

```suggestion
import ErrorIcon from '../../assets/images/icon-alert-danger-md.svg';
```